### DISCUSSION TITLE
使用搜图结果以私聊发送后，再将  搜索结果有多条时以聊天记录发送(群聊)改为true，将不再通过私聊发送而是群聊中发送

### DISCUSSION THREAD

**Discussion by WwwwGaa** [Tsuk1ko/CQ-picfinder-robot#416] [问题求助]
尝试过私聊改为true 然后把群聊的改成false但是私聊依旧是多条而非聊天记录

**Comment by Tsuk1ko** [comment]
确实有逻辑上的 bug，pmSearchResult 启用后需要同时启用 privateForwardSearchResult 和 groupForwardSearchResult 才行

**Comment by Tsuk1ko** [comment]
已修复
